### PR TITLE
Upper bound CRlibm 0.4 in all tagged versions of ValidatedNumerics

### PR DESCRIPTION
**Pull request by dpsanders**

### CODE REVIEW

**Comment by tkelman** [comment]
Looks right. What's broken?

**Comment by tkelman** [comment]
Wait no, there aren't any CRlibm versions available for Julia versions prior to 0.4, so adding this dependency would make those early ValidatedNumerics versions not installable on Julia 0.3 if anyone's still using that. Maybe better to upper-bound the Julia dependency to 0.4 for those early versions instead of adding an unsatisfiable dependency.

**Comment by dpsanders** [comment]
The upper bound is on the version of CRlibm.jl, not on the version of
julia. Or have I misunderstood?

**Comment by dpsanders** [comment]
Ah OK I see.

File: ValidatedNumerics/versions/0.0.3/requires [hunk 1]
```diff
@@ -1 +1,2 @@
 julia 0.0 0.3
+CRlibm 0.0 0.4
```
Comment (tkelman) [inline ValidatedNumerics/versions/0.0.3/requires]: 0.0.1, 0.0.2, and 0.0.3 won't need the CRlibm dependency

**Comment by dpsanders** [comment]
Thanks, think I've got it right now!

**Comment by dpsanders** [comment]
CRlibm v0.4 no longer automatically defines the functions that ValidatedNumerics requires.

File: ValidatedNumerics/versions/0.6.0/requires [hunk 2]
```diff
@@ -1,5 +1,5 @@
-julia 0.4
-CRlibm 0.2.2
+julia 0.4 
+CRlibm 0.3 0.4
```
Comment (tkelman) [inline ValidatedNumerics/versions/0.6.0/requires]: is raising the minimum also intended?

Comment (dpsanders) [inline ValidatedNumerics/versions/0.6.0/requires]: Yes, thanks.

**Comment by dpsanders** [comment]
Could this be merged, please?

**Pull request merged**